src/ai/backend/client/cli/vfolder.py

```diff
@@ -445,18 +445,23 @@ def cp(filenames):
     is_flag=True,
     help="Skip an error caused by file not found",
 )
-def mkdir(name, path, parents, exist_ok):
+def mkdir(name, paths, parents, exist_ok):
     """Create an empty directory in the virtual folder.
 
     \b
     NAME: Name of a virtual folder.
-    PATH: The name or path of directory. Parent directories are created automatically
+    PATH: Multiple arguments of name or path of directory seperated by space. Parent directories are created automatically
```

Comment:
This is a multi-value arguments, not argument-separated-by-space. They are different one: `"value1 value2 value3"` vs. `value1 value2 value3`.
Let's simply reveal that "you may specify multiple paths to create at once."